### ISSUE TITLE
Harmony HUB cannot be found

### ISSUE THREAD

**Issue by adx-git** [iobroker-community-adapters/ioBroker.harmony#59]
The adapter was working fine for a while, some days ago, without an obvious change, the adapter was not able to communicate with the hub.
Resetting the Adapter, restarting the server and reinstalling of the adapter did not help.
After reinstalling the adapter, the adapter was not able to find the hub.
Only after changing the subnet to the IP of the HUB, the HUB was found again.
But I still get error messages every 10 ~ 20 seconds.
`
 [DISCOVER] Discovered Harmony_Hub (192.168.3.21) and will try to connect
`
The adapter itself does not provide any helpful debug information.
Please let me know if there is any kind of test or logs I could provide.

OS: Linux
Adapter Version: 1.2.2 / 1.2.1

**Comment by Pmant** [comment]
When it comes to network configuration there is not really much the adapter could tell you. Could just be an unstable WIFI Connection or a blocking firewall. Does the Harmony APP has a persistent connection to the hub?

**Comment by adx-git** [comment]
Yes, there is no issue with the Harmony APP.
The error message is gone after reset of the harmony adapter, this is strange as I did not change anything in the network settings.
Still, if I use the default subnet, the hub cannot be found, only setting the subnet to the IP of the hub does the trick.

**Comment by pschyrembel258** [comment]
I have the same issue. Using the IP of my hub for the subnet worked for me either, but I have two hubs. The adapter only allows one instance, so I am not able to get both hubs in iobroker.

Iobroker is installed on a Synology NAS.

**Comment by mcm1957** [comment]
As this issue is ruther old, I will close here.
Please open a new issue if problem still exists and add all relevant release informations:
- adapter
- js-controller
- admin
- node
- o/s
